modules/standard/BigInteger.chpl

```diff
@@ -240,13 +184,22 @@ module BigInteger {
     @chpldoc.nodoc
     var localeId : chpl_nodeID_t;      // The locale id for the GMP state
 
+    /*
+      Initializes a :record:`bigint` to an initial value of 0.
```

Comment:
Should this `0` have double backticks? I notice most other literals you've added here do.